tests/lib/schema_validation_test.py

```diff
@@ -231,6 +231,73 @@ def test_over_maximum_total_vfs_is_invalid(
         with pytest.raises(js.ValidationError, match=str(invalid_values)):
             libnmstate.validator.validate(default_data)
 
+    @pytest.mark.parametrize("invalid_values", [-50, -1])
+    def test_below_minimum_vf_id_is_invalid(
+        self, default_data, invalid_values
+    ):
+        default_data[Interface.KEY][0].update(
+            {
+                Interface.TYPE: InterfaceType.ETHERNET,
+                Ethernet.SRIOV_SUBTREE: {
+                    Ethernet.SRIOV.TOTAL_VFS: 2,
+                    Ethernet.SRIOV.VFS_SUBTREE: [
+                        {Ethernet.SRIOV.VFS.ID: invalid_values}
+                    ],
+                },
+            }
+        )
+
+        with pytest.raises(js.ValidationError, match=str(invalid_values)):
+            libnmstate.validator.validate(default_data)
+
+    @pytest.mark.parametrize("valid_values", [0, 1, 20])
```

Comment:
How about `vf_id` for the argument name and generalize the test name to `test_valid_vf_ids`

Reply:
Seems fine! Thanks

Reply:
ping